html/partials/unmanaged_files.html.haml

```diff
@@ -0,0 +1,29 @@
+- if unmanaged_files && unmanaged_files.files.length > 0
+  %div
+    %a.scope_anchor{:id => "unmanaged_files"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace